exercise-3/styles.css

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.wrapper {
+  background-color: #413E3C;
+  display: flex;
+  flex-direction: row;
+  height: 30rem;
+  justify-content: center;
+  margin-top: 5rem;
+  width: 75rem;
+}
+
+input {
+  position: relative ;
+  top: 25rem;
+}
+
+labellll {
+  display: none;
+  position: fixed;
+}
+
+label {
+  padding-top: 5rem;
+  position: absolute;
+  left: -200rem;
+}
+
+input:checked + label {
+  display: block;
+  transition: 0.25s;
```

Comment:
The default easing curve isn’t linear in most browsers, so this isn’t bad, but we can be more expressive here! This is where the “personality” comes in.

The [animation pages](https://material.io/design/motion/understanding-motion.html#usage) of Google’s Material design manifesto not only provide some good `cubic-bezier()` curves (they’re in [Fondant](https://github.com/manifoldco/fondant/blob/master/src/theme.js#L146) btw!), but some important questions to ask when selecting a timing/easing:

- How large is this element—should it feel weighty, or light as a feather?
- Will this element reappear later? Should it animate from far off, or just off-screen?
- How is it moving? Is physics applying to it (translation, scale)? Or is it a light-based animation (color, opacity)?

Answering these questions can help us better lock in a timing and easing.